src/Sarif.Converters/Sarif.Converters.csproj

```diff
@@ -40,14 +40,19 @@
   <Choose>
     <When Condition="$(TargetFramework) == 'netstandard2.0'">
       <ItemGroup>
-        <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+        <!-- We have to ship pre-patch versions of NewtonSoft for VisualStudio SDK. 
```

Comment:
>NewtonSoft [](http://example.com/codeflow?start=51&length=10)

Actually, for this case, v9.0.1 is required to maximize down-level compatibility for this assembly.  Specifically, v9 is the first newtonsoft version to support .NET standard 1.0.